pex/bin/pex.py

```diff
@@ -935,12 +934,9 @@ def to_python_interpreter(full_path_or_basename):
             )
 
             for resolved_dist in resolveds:
-                log(
-                    "  %s -> %s" % (resolved_dist.requirement, resolved_dist.distribution),
-                    V=options.verbosity,
-                )
```

Comment:
Why remove this?

Reply:
We no only have direct requirements, so for `pex requests -orequests.pex` we could only log:
```
pex: Resolving distributions ([requests])
pex:   requests -> ~/.pex/installed_wheels/....
```
And that - to a close reader - looks alarming if they know requests has 5-10 transitive dependencies - which it does.
To make this un-alarming the log would need to change to include all the dists that satisfy each top-level requirement. That's not hard, but its also a blob of code that would produce logs that ~only I ever read ... and this is not useful information. Its contained in PEX-INFO anyhow.